src/libraries/System.Formats.Cbor/tests/Writer/CborWriterTests.Tag.cs

```diff
@@ -158,7 +158,7 @@ public static void WriteInteger_SingleValue_HappyPath(string valueString, string
         [InlineData("1", "c4820001")]
         [InlineData("-1", "c4820020")]
         [InlineData("1.1", "c482200b")]
-        [InlineData("1.000", "c4820001")]
+        [InlineData("1.000", "c482221903e8")]
```

Comment:
This might warrant more investigation. Based on what I remember @eiriktsarpalis saying this was being parsed as `1000` implying the trailing zeros are being handled incorrectly?

Reply:
Actually this is just a revert of the change here: https://github.com/dotnet/runtime/pull/47666/files#r568680279

Reply:
Ah, I was misremembering. It's `1000e-3` which is still `1` and these are for `decimal` so I think everything is ok...